.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 19.10b0
+    rev: 20.8b1
     hooks:
     - id: black
 -   repo: https://github.com/thlorenz/doctoc
-    rev: master
+    rev: v2.0.0
```

Comment:
what's the motivation for the version changes?

Reply:
sorry, it wasn't my intention to change this file, I undid the changes